subsys/bluetooth/audio/vocs.c

```diff
@@ -46,6 +46,20 @@ static void location_cfg_changed(const struct bt_gatt_attr *attr, uint16_t value
 	BT_DBG("value 0x%04x", value);
 }
 
+static ssize_t read_location(struct bt_conn *conn, const struct bt_gatt_attr *attr,
+			     void *buf, uint16_t len, uint16_t offset)
+{
+	struct bt_vocs *inst = attr->user_data;
+
+	BT_DBG("0x%02x", inst->srv.location);
+	return bt_gatt_attr_read(conn, attr, buf, len, offset, &inst->srv.location,
+				 sizeof(inst->srv.location));
+}
+
+static void output_desc_cfg_changed(const struct bt_gatt_attr *attr, uint16_t value)
+{
+	BT_DBG("value 0x%04x", value);
+}
```

Comment:
The move of these functions are quite unnecessary, and could be omitted from the PR. 

What was the issue you found in this file anyhow?